packages/augur-sdk/src/api/ZeroX.ts

```diff
@@ -329,10 +329,15 @@ export class ZeroX {
   }
 
   async addOrder(order) {
-    if (this.browserMesh) {
-      return this.browserMesh.addOrdersAsync([order]);
-    } else {
-      return this.meshClient.addOrdersAsync([order]);
+    try {
+      if (this.browserMesh) {
+        return this.browserMesh.addOrdersAsync([order]);
+      } else {
+        return this.meshClient.addOrdersAsync([order]);
+      }
+    }
+    catch(error) {
+      return setTimeout(this.addOrder(order), 5000);
```

Comment:
👍

Reply:
This needs to actually detect the error and probably do some sort of back-off with a limit or we can get into a terrible busy loop here.